common.mk

```diff
@@ -26,3 +26,136 @@ Q_LIPO       = $(if $(V),,@echo "LIPO   $(@F)";)
 Q_SOM        = $(if $(V),,@echo "BINDINGTOOLSFORSWIFT $(NAME)";)
 Q_XCODEBUILD = $(if $(V),,@echo "XCODEBUILD/$* $(NAME)";)
 Q_SWIFTC     = $(if $(V),,@echo "SWIFTC $(@F)";)
+
+# namespaces
+COMMON_NAMESPACES= \
```

Comment:
Minor - since you're putting this into a global include, it's best to now make these identifiers self-identify more. Suggest `TYPE_O_MATIC_COMMON_NAMESPACES` and `TYPE_O_MATIC_IOS_NAMESPACES` and ``TYPE_O_MATIC_MAC_NAMESPACES`

Reply:
Done